src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+
+import command.AddCommand;
+import command.Command;
+import command.DeleteCommand;
+import command.DoneCommand;
+import command.ListCommand;
+import command.ByeCommand;
+import command.FindCommand;
+import command.ClearCommand;
+
+/**
+ * Parser class to parse a given input command.
+ *
+ */
+public class Parser {
+
+    /**
+     * To parse the given user command to be recognized.
+     *
+     * @param fullCommand full command input by user.
+     * @return A Command which is able to be executed.
+     * @throws DukeException If command is invalid.
+     */
+    public static Command parse(String fullCommand) throws DukeException {
+        String[] arr = fullCommand.split(" ", 2);
+        String cmd = arr[0];
+        Command command;
+        switch (cmd) {
+        case "todo":
```

Comment:
According to the Java Coding Standard,
> The explicit `//Fallthrough` comment should be included whenever there is a case statement without a break statement.

This affects line 31 - 32.

Reply:
These cases actually have break statement. This is a syntax indicating that they will all execute the code after the event case.